devices/client.go

```diff
@@ -0,0 +1,57 @@
+package devices
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/tidepool-org/devices/api"
+	"go.uber.org/fx"
+	"google.golang.org/grpc"
+
+	"github.com/tidepool-org/platform/errors"
+)
+
+var ClientModule = fx.Provide(NewClient)
+
+type Config struct {
+	DevicesClientAddress string `envconfig:"TIDEPOOL_DEVICES_CLIENT_ADDRESS" required:"true"`
```

Comment:
This is nice!

Reply:
We can finally search for environment variables!